### ISSUE TITLE
Confirmação de pagamento (BUG)

### ISSUE THREAD

**Issue by sidneydevelop** [paghiper/whmcs#38]
Após emitir a cobrança via boleto, aguardo alguns minutos e o sistema está confirmando o pagamento automaticamente no valor R$0,00 sem mesmo o pagamento ter sido efetuado.

**Comment by henriqueccruz** [comment]
Olá @sidneydevelop, a fatura está sendo marcada como paga ou é apenas uma transação que aparece na lista?

**Comment by sidneydevelop** [comment]
É marcada como paga e o cliente recebe e-mail que foi pago no valor informado acima.

**Comment by henriqueccruz** [comment]
Vou precisar de mais alguns dados pra checar isso por favor.
Me fale qual versão do plug-in está usando, seu e-mail de cadastro e o ID da fatura baixada por favor. Apagaremos a mensagem assim que pegarmos os dados.

**Comment by henriqueccruz** [comment]
@sidneydevelop - pega a versão atual do repositório novamente por favor.
Consegui replicar e o último commit resolve isso.

**Comment by henriqueccruz** [comment]
Sim, fiz um novo commit, algumas horas atras resolvendo este problema.
